chapter06/red/exercise1.pl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/perl
+use warnings;
+use strict;
+use Storable qw( nstore retrieve );
+
+my $db = '.coconet.db';
+my %stats;
+%stats = %{ retrieve $db } if -f $db;
+
+my $fn = shift @ARGV // 'coconet.dat';
+open my $fh, '<', $fn;
```

Comment:
But my users like seeing `readline() on closed filehandle $fh at ./exercise1.pl line 13`... (alright, I'll bother with error handling in upcoming exercises).